.github/workflows/autoDIR.yml

```diff
@@ -0,0 +1,29 @@
+name: autoDIR
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version: [3.7]
+    steps:
+
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Update DIRECTORY
+        run: |
+          scripts/build_directory_md.py 2>&1 | tee DIRECTORY.md
+          git config --global user.name 'autoDIR'
+          git config --global user.email 'mrvnmchm@users.noreply.github.com'
+          git remote set-url origin https://x-access-token:${{ secrets.gh_token }}@github.com/$GITHUB_REPOSITORY
+          git checkout $GITHUB_HEAD_REF
+          git diff-files --quiet 
```

Comment:
I believe that if this command fails then the entire PR is marked with a failing test ❌ which we do not want.

On line 28, use `fixup:` instead of `Update:` because when a merge happens git will autosquish that commit into the previous commit.

Put a `\n` at the end of the file so that GitHub does not put 🚫 at the end of the file.

Reply:
That's correct @cclauss, the `git diff-files --quiet` commands exit code is the same as for git diff-index (0 == no differences; 1 == differences). In this case, if failure, it will continue with the push. Would you have any suggestion for evaluating if the DIRECTORY.md file has changed?

I will make the changes to line 28, and add the newline at the end in upcoming PR